firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -318,6 +322,12 @@ public Task<HttpURLConnection> createRealtimeConnection() {
                         "Firebase Installations failed to get installation auth token for real-time.",
                         installationAuthTokenTask.getException()));
               }
+              if (!installationIdTask.isSuccessful()) {
+                return Tasks.forException(
+                    new FirebaseRemoteConfigClientException(
+                        "Firebase Installations failed to get installation ID for real-time.",
```

Comment:
Thoughts on using "config update listener connection" here instead of "real-time"? This error comes up fairly frequently for fetch since it's the first network request made, so tying it more closely to the API method names might make debugging easier.

(Same with the auth token error message, too)

Reply:
updated